src/olympia/blocklist/management/commands/export_blocklist.py

```diff
@@ -14,42 +15,22 @@ class Command(BaseCommand):
 
     def add_arguments(self, parser):
         """Handle command arguments."""
-        parser.add_argument('id', help='CT baseline identifier', metavar=('ID'))
         parser.add_argument(
-            '--addon-guids-input',
-            help='Path to json file with [[guid, version],...] data for all '
-            'addons. If not provided will be generated from '
-            'Addons&Versions in the database',
-            default=None,
```

Comment:
I wanted to ask what was the use case for that. I think we should modify to give a timestamp and load from storage instead of passing arbitrary lists for blocked/unblocked. or just always generate from db.

Reply:
I think it was mainly used while mlbf were a new and exotic thing, and I needed to manually generate bloom filters to double check everything was being created as expected, without the added potential variance coming from building the values from the database.  These days the library and implementation is pretty stable and know so I guess not really needed?

Reply:
Removed for now.. we can update later